src/infra/core/Cargo.toml

```diff
@@ -32,15 +32,16 @@ query-extensions-json = ["dep:datafusion-functions-json"]
 
 [dependencies]
 # Kamu
-internal-error = { workspace = true }
 container-runtime = { workspace = true }
-kamu-data-utils = { workspace = true }
-opendatafabric = { workspace = true }
+internal-error = { workspace = true }
 kamu-accounts = { workspace = true }
-kamu-datasets = { workspace = true }
+kamu-auth-rebac = { workspace = true }
 kamu-core = { workspace = true }
+kamu-data-utils = { workspace = true }
+kamu-datasets = { workspace = true }
```

Comment:
Similarly, can we avoid depending on this domain?

Reply:
> kamu-datasets = { workspace = true }

is the `kamu_datasets::{DatasetEnvVar, DatasetKeyValueService};` dependencies for `FetchService` -- I haven't touched this area

what I did: sorting the imports